oci/README.md

```diff
@@ -0,0 +1,3 @@
+# Overview
```

Comment:
To be consistent with Azure and AWS we can add a link to our section in the main README https://github.com/DataDog/datadog-serverless-functions/blob/master/README.md